fetch/api/redirect/redirect-count-cross-origin.js

```diff
@@ -0,0 +1,59 @@
+if (this.document === undefined) {
+  importScripts("/resources/testharness.js");
+  importScripts("../resources/utils.js");
+  importScripts("/common/utils.js");
+  importScripts("/common/get-host-info.sub.js");
+}
+
+function redirectCount(desc, redirectUrl, redirectLocation, redirectStatus, maxCount1, maxCount2, requirePreflight)
+{
+  var allow_headers = "";
+  var requestInit = {"redirect": "follow"};
+  if (requirePreflight) {
+      requestInit.headers = [["x-test", "test"]];
+      allow_headers = "&allow_headers=x-test";
+  }
+
+  var uuid_token = token();
+
+  var urlParameters = "?token=" + uuid_token + "&max_age=0";
+  urlParameters += "&redirect_status=" + redirectStatus;
+  urlParameters += "&max_count=" + maxCount1;
+  urlParameters += allow_headers;
+  urlParameters += "&location=" + encodeURIComponent(redirectLocation + "?token=" + uuid_token + "&max_age=0&max_count=" + (maxCount1 + maxCount2) + allow_headers);
```

Comment:
It seems you're adding allow_headers and max_age twice. Also max_age is never used in the Python resource.